firefox_ui_tests/functional/locationbar/test_star_in_autocomplete.py

```diff
@@ -45,15 +49,18 @@ def test_star_in_autocomplete(self):
         with self.marionette.using_context('content'):
             self.marionette.navigate(self.marionette.absolute_url('layout/mozilla_grants.html'))
```

Comment:
Ok, I've done this. I wasn't sure whether I should loop through the list of urls visiting them, or just use `self.test_urls[0]` as we know there is only one. It feels better to loop through the list even though it is unnecessary, as that will continue to work if for some reason we add more urls to the test later. You'll see that change in the next commit, coming soon.